ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -24,6 +25,7 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
         readonly IConnectableObservable<T> _source;
         IDisposable _inner;
                 
+        private int _activated;
```

Comment:
@kentcb Oh, I only saw an overload with `T`on MSDN, but didn't see it was restricted to reference types, my fault!